networkx/algorithms/approximation/clique.py

```diff
@@ -40,6 +40,14 @@ def maximum_independent_set(G):
     iset : Set
         The apx-maximum independent set
 
+    Examples
+    --------
+    >>> import networkx as nx
+    >>> from networkx.algorithms.approximation.clique import maximum_independent_set
+    >>> G = nx.path_graph(10)
+    >>> maximum_independent_set(G)
```

Comment:
Similar to the other PRs you've opened related to import styles, we want to encourage users to access the functions from the proper namespace:

```suggestion
    >>> G = nx.path_graph(10)
    >>> nx.approximation.maximum_independent_set(maximum_independent_set(G))
```